src/cli/user/register.rs

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::drawbridge::{client, UserSpec};
+
+use anyhow::Context;
+use clap::Args;
+use drawbridge_client::types::UserRecord;
+use drawbridge_client::Url;
+use oauth2::AccessToken;
+use openidconnect::core::{CoreClient, CoreProviderMetadata, CoreUserInfoClaims};
+use openidconnect::ureq::http_client;
+use openidconnect::{ClientId, IssuerUrl};
+
+/// Register a new user account with a package host.
+#[derive(Args, Debug)]
+pub struct Options {
```

Comment:
We should really have `$XDG_CONFIG_HOME/enarx/conf.toml`, where `oidc_domain` and `oidc_client_id` would be configured.

Reply:
I've filed https://github.com/enarx/enarx/issues/2021 . We can pursue this for a future release.